### PR TITLE
fix(debug command): update err msg `int` -> `integer`

### PR DESCRIPTION
**Pull request by AntiTopQuark**
https://github.com/apache/kvrocks/issues/2045

### CODE REVIEW

**Review by mapleFU — DISMISSED** [review]
+1

**Comment by mapleFU** [comment]
Seems lint failed, would you mind run `./x.py format` or using `clang-format` to format the edited file?

**Comment by AntiTopQuark** [comment]
> Seems lint failed, would you mind run `./x.py format` or using `clang-format` to format the edited file?

done, thank you very much

File: src/common/status.h [hunk 1]
```diff
@@ -168,7 +168,7 @@ struct StringInStatusOr<T, std::enable_if_t<sizeof(T) < sizeof(std::string)>> :
   StringInStatusOr(StringInStatusOr<U>&& v) : BaseType(new std::string(*std::move(v))) {}  // NOLINT
   template <typename U, typename std::enable_if_t<!StringInStatusOr<U>::inplace, int> = 0>
   StringInStatusOr(StringInStatusOr<U>&& v)  // NOLINT
-      : BaseType((typename StringInStatusOr<U>::BaseType &&)(std::move(v))) {}
+      : BaseType((typename StringInStatusOr<U>::BaseType&&)(std::move(v))) {}
```
Comment (mapleFU) [inline src/common/status.h]: cc @PragmaTwice 🤔 why would this file always get format?

Comment (mapleFU) [inline src/common/status.h]:
```suggestion
      : BaseType((typename StringInStatusOr<U>::BaseType &&)(std::move(v))) {}
```

Comment (git-hulk) [reply src/common/status.h]: This was caused by clang-format-16 behaving differently from clang-format-14, that said clang-format-16 will remove the space between the type and &&, but clang-format-14 won't do that.

**Review by git-hulk — APPROVED** [review]

**Pull request merged**